### DISCUSSION TITLE
Is there a way to assign value to doc header? (View -> Page Layout)

### DISCUSSION THREAD

**Discussion by zitotrex** [jxlsteam/jxls#341] [Q&A] [answered]
A jx:area comment goes in the normal spreadsheet area, was wondering if we could also assign a value to the page layout -> doc header somehow

i tried ${data[0].CODE} which predictably failed

**Comment by SoltauFintel** [comment, marked as answer]
Use Apache POI for changing the page layout.

If you use an older version you should try 3.0 because page layout settings have been improved.

**Comment by zitotrex** [comment]
I understand, thanks.

I am only using the JXLS Template with Report Server at this time, don't think I can mix my template with Apache POI. Oh well.
